docs/spec/spec.md

```diff
@@ -159,6 +159,11 @@ spec:
     metadata: ...
       labels:
         knative.dev/type: "function"  # One of "function" or "app"
+      annotations:
+        # +optional. When not specified, the revision can scale down to 0 pods
+        autoscaling.knative.dev/minReplicaCount: "2"  
+        # +optional. When not specified, there's no upper scale bound
+        autoscaling.knative.dev/maxReplicaCount: "10"  
```

Comment:
This is a feature of the KPA implementation that we bundle today (once you implement it?), so I'd rather not document this here because it is not within the scope of conformance (other KPA implementations may choose to ignore these).

This feels like it belongs in documentation for our KPA plugin.  cc @josephburnett 

Ack that this feels a bit crazy until we can actually swap in different autoscalers :)

Reply:
Addressed this above in https://github.com/knative/serving/pull/1980/files#r218251531.

I think we should leave this out of the spec.  We propagate all annotation from Service -> Configuration -> Revision so you can use this knob at whatever level you want.  But the entities along that path should not necessarily be aware of, or couple with, those annotations.  Min/max is an implementation specific knob.